src/main/java/net/smartcosmos/dao/objects/converter/ObjectResponseToObjectEntityConverter.java

```diff
@@ -0,0 +1,42 @@
+package net.smartcosmos.dao.objects.converter;
+
+import net.smartcosmos.dao.objects.domain.ObjectEntity;
+import net.smartcosmos.dto.objects.ObjectResponse;
+import net.smartcosmos.security.user.SmartCosmosUser;
+import net.smartcosmos.security.user.SmartCosmosUserHolder;
+
+import net.smartcosmos.util.UuidUtil;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.format.FormatterRegistrar;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author voor
```

Comment:
I personally dislike the `@author` annotation.  It provides no more useful information than can be acquired from blame and implies an owner who may have long since moved on from "owning" or "authoring" a particular class.  However, if we are going to use them make sure it is the correct person.